simulation_parameters/Constants.cs

```diff
@@ -603,11 +603,18 @@ public static class Constants
     /// </summary>
     public const float PILUS_INVULNERABLE_TIME = 0.25f;
 
+    /// <summary>
+    ///   How much larger a cell has to be to ignore the pilus collision of another on contact.
+    /// </summary>
+    public const float PILUS_INEFFECTIVE_RATIO = 2.5f;
```

Comment:
Wouldn't it be better to have a progressively less effective pilus with the size difference? Rather than just have a one flat out cutoff where you just can't deal with damage. I can imagine a bunch of scenarios where dealing exactly 0 damage is not going to feel good.

Reply:
I may have miscommunicated. The pilus always still does damage. It's just that cells can be engulfed despite the presence of a pilus if the engulfer is significantly larger than them. They still cause damage.

This is to address some complaints about predator builds being pretty frustrating once pili become wide-spread, and to make pili less effective against what should be more significant threats. A single bacterium shouldn't be able to tear apart a eukaryote with zero effort.

That being said, I do worry about communicating this, as players are going to be in for a nasty surprise if they attempt to poke a massive cell with a pilus expecting to kill them without issue like before... And of course, the exact cutoff may be unclear to the player.